LiskHQ/lisk-sdk
# Save tried peers in db - Closes #4358

### What was the problem?
Connected peers was saved in the cleanup instead of periodically saving the tried peers in the database.

### How did I solve it?
- Expose tried peers from the p2p
- Update to save the tried peers instead of connected peers
- Set to save tried peers every 10min

### How to manually test it?
- Observe after 10min, all of the tried peers is saved in database
- on startup, it should use the list

### Review checklist

- [ ] The PR resolves #4358
